drivers/SmartThings/zwave-switch/src/test/test_zooz_zen_30_dimmer_relay.lua

```diff
@@ -64,17 +64,6 @@ test.register_coroutine_test(
       mock_zooz_zen_30_dimmer_relay.id,
       { capability = "refresh", component = "main", command = "refresh", args = {} }
     })
-    test.socket.zwave:__expect_send(
```

Comment:
This seems to be a copy paste error, and can be removed safely. This test is testing that this command is sent to the device, but it was erroneously expecting that it be sent twice.

Reply:
this is sending to the unencapsulated device though

Reply:
Im not sure I understand. The [handler](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/494974b8c37e1ab3274001b8d43ebe2e7083ca80/drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua#L110-L114) only sends 3 commands. Why would we expect a fourth in the test?

Reply:
oh I see it was a duplicate now